src/components/common/ui/form/radio/RadioGroup.stories.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
+import { Box } from '@mui/material';
+import type { Meta, StoryObj } from '@storybook/react';
+import { Form, Formik } from 'formik';
+
+import RadioGroup from '@/components/common/ui/form/radio/RadioGroup';
+
+type RadioGroupStoryType = {
+  name: string;
+  options: { label: string; value: string }[];
+};
+
+const RadioGroupStory: React.FC<RadioGroupStoryType> = (
+  args: RadioGroupStoryType,
+) => {
+  const [name, setName] = useState('Name');
```

Comment:
This is never used. Make use of this or other hook so the buttons in storybook were clickable.